### ISSUE TITLE
Add Support To Rank Math SEO plugin

### ISSUE THREAD

**Issue by isaumya** [benhuson/wp-subtitle#47]
First of all thanks a lot for this awesome and amazing plugin. But I think in the SEO world Rank Math is also a great plugin. I have used it in some of my client sites and I am super satisfied with it. I started to prefer it over Yoast. So, I think it would be great id you also add the Rank Math support.

**Comment by benhuson** [comment]
https://wordpress.org/plugins/seo-by-rank-math/

**Comment by isaumya** [comment]
@benhuson Yes that's the plugin I am talking about. In the plugin description you have said: 

> YOAST SEO AND SEOPRESS PLUGIN SUPPORT
The plugin allows you to include the subtitle in your meta titles and descriptions via the Yoast SEO and SEOPress plugins.
Similar to the Yoast %%title%% placeholder which inserts the post title, you can use %%wps_subtitle%%.
There are also addition placeholders and filters to allow to to customize seperators for the subtitle.

So, I'm just asking you to add support for Rank Math as well.
